packages/stark-rbac/tslint.json

```diff
@@ -1,5 +1,11 @@
 {
-  "extends": ["tslint:latest", "tslint-sonarts", "codelyzer", "tslint-config-prettier", "../stark-build/config/tslint.json"],
+  "extends": [
+    "tslint:latest",
+    "tslint-sonarts",
```

Comment:
I'm wondering why we don't expose SonarTS through code-style. Or is it because we do not customize it? 🤔

Reply:
Well, it is because SonarTS is a choice we made at the NBB and it is an additional set of rules on top of the ones of TSLint but they are not mandatory. So we don't want to enforce anyone to use such package if they don't want to.

Moreover, adding a "subpackage" like this will make the structure of our configs a lot more difficult to maintain in the `code-style` package (for example, look at what we do for [TypeScript config + Angular config](https://github.com/NationalBankBelgium/code-style/tree/master/lib/tsconfig) since we had no choice because Angular is crucial at the NBB).

Reply:
Having `tslint-sonarts` in our code-style tslint config would imply that we would have to add a peer dependency to the package. We prefer to keep code-style package without any dependency 😊

Reply:
Ok, but we do do that for codelyzer right?

Reply:
Indeed, but codelyzer is considered a "must" in the Angular ecosystem whereas SonarTS is not... it is encouraged in any Typescript application... so that's why I suggest we don't include it in our code-style for now